asdf/tests/test_compression.py

```diff
@@ -90,3 +90,9 @@ def test_bzp2(tmpdir):
     tree = _get_large_tree()
 
     _roundtrip(tmpdir, tree, 'bzp2')
+
+
+def test_lz4(tmpdir):
```

Comment:
This introduces `lz4` as a test dependency. The tests are failing because this is not currently accounted for. Can you please update `setup.py` so that this dependency is resolved and the tests pass?